tests/autostep-cli.tests/CommandInvocationTests.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.CommandLine;
+using System.CommandLine.Builder;
+using System.CommandLine.Parsing;
+using System.Threading;
+using Xunit;
+using FluentAssertions;
+using static AutoStep.CommandLine.CreateProjectCommand;
+using FluentAssertions.Primitives;
+
+namespace AutoStep.CommandLine.Tests
+{
+    public class CommandInvocationTests
+    {
+        private static readonly CreateProjectDelegate _successProjectCreation = MakeCreateProjectDelegateFromStatusCode(0);
```

Comment:
The underscores in the variable names here aren't consistent with the rest of the project.